index.js

```diff
@@ -1,17 +1,33 @@
 const c = require('compact-encoding')
 const definition = require('./spec/hyperdb')
 const schema = require('./spec/hyperschema')
-
-const addMailboxEncoding = {
-  requestEncoding: schema.resolveStruct('@blind-peer/request-mailbox'),
-  responseEncoding: schema.resolveStruct('@blind-peer/response-mailbox')
-}
+const hypCrypto = require('hypercore-crypto')
+const b4a = require('b4a')
+const Hypercore = require('hypercore')
 
 const postEncoding = {
   requestEncoding: schema.resolveStruct('@blind-peer/request-post'),
   responseEncoding: c.none
 }
 
+const seedsEncoding = schema.resolveStruct('@blind-peer/seeds')
+
+function createMailbox (blindWriterEncryptionPublicKey, { entropy, autobaseKey, blockEncryptionKey } = {}) {
```

Comment:
TODO: this needs to use a proper versioned encoding instead of the current approach